packages/medusa/src/models/product.ts

```diff
@@ -21,7 +21,7 @@ import { ShippingProfile } from "./shipping-profile"
 import { SoftDeletableEntity } from "../interfaces/models/soft-deletable-entity"
 import { generateEntityId } from "../utils/generate-entity-id"
 
-export enum Status {
+export enum ProductStatus {
```

Comment:
suggestion(non-blocking): @olivermrbl we should standardise the approach on type, enum declaration, sometime they are put in the type directory sometime they are in the model. Should we put them in the type directory to keep all related types to a domain close together?